core/trino-main/src/main/java/io/trino/server/ui/FormUiAuthenticatorModule.java

```diff
@@ -37,10 +37,10 @@ public FormUiAuthenticatorModule(boolean usePasswordManager)
     @Override
```

Comment:
Password authentication manager should also be binded conditionally in `ServerSecurityModule`